*** ./smali/android/view/ViewDebug.smali	Tue Apr 28 13:28:30 2015
--- /home/builder/patchrom-v6/d855/temp/new_smali/framework.jar.out/./smali/android/view/ViewDebug.smali	Tue Apr 28 13:36:35 2015
***************
*** 4688,4693 ****
--- 4688,4701 ----
      const-string v3, "\\n"
  
      invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
+     const-string v2, "\r"
+ 
+     const-string v3, "\\r"
+ 
+     invoke-virtual {v1, v2, v3}, Ljava/lang/String;->replace(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
      :try_end_0
      .catchall {:try_start_0 .. :try_end_0} :catchall_0
  
