*** ./smali/com/android/server/input/InputManagerService$WindowManagerCallbacks.smali	Tue Apr 28 13:26:42 2015
--- /home/builder/patchrom-v6/d855/temp/new_smali/services.jar.out/./smali/com/android/server/input/InputManagerService$WindowManagerCallbacks.smali	Tue Apr 28 13:34:40 2015
***************
*** 33,38 ****
--- 33,41 ----
  .method public abstract notifyANR(Lcom/android/server/input/InputApplicationHandle;Lcom/android/server/input/InputWindowHandle;Ljava/lang/String;)J
  .end method
  
+ .method public abstract notifyBackLidSwitchChanged(JZ)V
+ .end method
+ 
  .method public abstract notifyConfigurationChanged()V
  .end method
  
